label.py

```diff
@@ -17,12 +17,15 @@
 
 images_to_show = [i for i in utils.get_images() if i not in labels]
 random.shuffle(images_to_show)
+images_marked = [i for i in utils.get_images() if i in labels]
+images_to_show = images_marked + images_to_show
```

Comment:
We should add an option to either only show unmarked images or show all images.

Reply:
Presently it starts with the very next image you would have started labeling. If you press 'a' (to go left) it will show all the previously marked images. 
I can add it as an argparse interface if you feel it is necessary!

Reply:
Yeah it's a bit confusing, I think it's better to add two modes

Reply:
Okay! I will add two modes. One only for unmarked images, and other for all images including unmarked images?

Reply:
Maybe one for unmarked and one for marked? It sounds even better (in one case, you only want to verify something you or somebody else has marked, in the other case you want to mark new images).

Reply:
Just to confirm, user will be able to edit his old labels. 
So in case of marked, user will be able to see his previous markings and edit them. He won't be allowed to mark new images.
In case of unmarked, user will be able to mark new images and change only the images he has marked in that run.

Reply:
Exactly!